src/enzyme_ad/jax/Passes/AffineRaising.cpp

```diff
@@ -0,0 +1,462 @@
+//===- AffineRaising.cpp - Raise affine loops to stablehlo ----------------===//
```

Comment:
can we call this stablehlo-raising instead of affine raising (the latter implies raise to affine)

Reply:
I based the naming on the existing arith-raise pass which raises from arith to stablehlo. I can rename to something like raise-affine-to-stablehlo ?

Reply:
yeah that would be great (and indeed arith raise is different from the other raising ones we're building since it has multiple different outputs, mhlo vs stablehlo)